crates/wash-cli/tests/fixtures/integrated-wkg-extended/wasmcloud.toml

```diff
@@ -0,0 +1,22 @@
+name = "ponger-config-component"
+language = "rust"
+type = "component"
+
+[component]
+wasm_target = "wasm32-wasip2"
+
+[[registry.pull.sources]]
+target = "wasmcloud:bus"
+source = "oci://ghcr.io/wasmcloud/interfaces"
```

Comment:
Should also pin a version here too

Reply:
So supporting pinning a specific version would be great, but there isn't actually support for that upstream, AFAIK, am I wrong about that? The usage here is going through `wkg` as normal, and [`PackageRef`](https://docs.rs/wasm-pkg-common/0.8.5/wasm_pkg_common/package/struct.PackageRef.html) has no way to specify a version actually... 

Obviously, it's trivial to add a `?tag=x.x.x` query param or something like that to enable the support on our side (I don't think there's a standard for this?), but even if we did, we need upstream support to be able to actually narrow this down, no?

Reply:
Sorry yep, you're right. The main wkg client will use this as the base repo and then calculate version from there